<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArmWidget</class>
 <widget class="QWidget" name="CArmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>600</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>测量3</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelReferencePicture">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>280</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/sketch.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>580</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>测量2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenPicture">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_1">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>610</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>参考0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>470</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>参考1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>290</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>计算角度</string>
   </property>
   <widget class="QLabel" name="label_line1_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>直角边1：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_line1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_line2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_line2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>直角边2：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>计算结果：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result2_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate2">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>480</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/btn.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>600</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>测量4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>620</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>测量5</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>580</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>C臂俯仰：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_Ry">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>C臂翻转：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_Rz">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>C臂偏转：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_Rx">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>520</y>
     <width>231</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>布拉门赛特线：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_B"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>踝间距：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_ankle"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowserMeasureValue">
   <property name="geometry">
    <rect>
     <x>1113</x>
     <y>210</y>
     <width>127</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1031</x>
     <y>171</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);


</string>
   </property>
   <property name="text">
    <string>参考长度：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditReferenceValue">
   <property name="geometry">
    <rect>
     <x>1113</x>
     <y>171</y>
     <width>127</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1031</x>
     <y>249</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
</string>
   </property>
   <property name="text">
    <string>直线夹角：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserMeasureValue_Angle">
   <property name="geometry">
    <rect>
     <x>1113</x>
     <y>249</y>
     <width>127</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1031</x>
     <y>210</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235, 235, 235);
</string>
   </property>
   <property name="text">
    <string>测量值：</string>
   </property>
  </widget>
  <zorder>labelReferencePicture</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>pushButtonOpenPicture</zorder>
  <zorder>radioButton_1</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>radioButton_4</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>pushButton_calculate2</zorder>
  <zorder>radioButton_5</zorder>
  <zorder>radioButton_6</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
